pkgs/top-level/all-packages.nix

```diff
@@ -4400,6 +4400,7 @@ with pkgs;
 
   curl = curlMinimal.override ({
     idnSupport = true;
+    zstdSupport = true;
```

Comment:
I am not sure how widely zstd is used in the wild. Would that be something we would want to have enabled by default?

Reply:
https://archlinux.org/packages/core/x86_64/curl/ - ArchLinux already used by default.